lib/ansible/modules/remote_management/manageiq/manageiq_database.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Alex Braverman Masis <alexbmasis@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: manageiq_database
+description:
+    - Manage appliance database configuration using the appliance_console_cli on ManageIQ.
+short_description: Configure ManageIQ appliance database.
+version_added: "2.4"
+author: "Alex Braverman Masis (@abraverm)"
+requirements:
+    - appliance_console_cli
+options:
+    internal:
+        description: Should ManageIQ appliance database be self hosted (internal).
+        default: True
+        type: bool
+    region:
+        description: ManageIQ region the database assosieted with.
+        default: 0
+    dbdisk:
+        description: |
+            In case of internal database, path of the disk to use for database
+            server storage.
+    username:
+        description: User name to use for ManageIQ database.
+        default: root
+    password:
+        description: Password to use for ManageIQ database.
+        default: smartvm
+    hostname:
+        description: ManageIQ database server IP or FQDN connect to.
+        default: 127.0.0.1
+    dbname:
+        description: Name of the database to use to store ManageIQ tables.
+        default: vmdb_production
+    backup_location:
+        description: Two types of locations supported by ManageIQ, local and remote.
+        default: local
+    backup_path:
+        description: File path or URI of the backup.
+        default: /tmp/backup
+    backup_username:
+        description: User name for remote backup location access.
+        default: root
+    backup_password:
+        description: Password for remote backup location access.
+        default: smartvm
+    pg_data_path:
+        description: When removing database, the files are removed too.
```

Comment:
Not exactly a description of this param.  Is this stating a fact, and the param just sets the path?  Or can you inhibit this behavior by unsetting this?